provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1534,9 +1534,9 @@ func generateSchema() schema.PackageSpec {
 			},
 			"eks:index:AccessEntryType": {
 				ObjectTypeSpec: schema.ObjectTypeSpec{
-					Type:        "string",
```

Comment:
I suspect there is no lint in this repo? Sadness. Do we track installing it already?

Reply:
Let me create a tracking issue and a follow-up PR for this!

Reply:
#1185